lib/renuo_cms_rails/configuration.rb

```diff
@@ -0,0 +1,29 @@
+module RenuoCmsRails
+  class << self
+    attr_writer :configuration
```

Comment:
good point. actually i was too lazy to rewrite that (see https://github.com/sgruhier/foundation_rails_helper), but maybe we should...?